### PR TITLE
Modifications to Simulation Environment

### PR DESCRIPTION
**Pull request by LamarMoore**
Changes:
- More detailed instructions for users 
- Mcstas manuals added to repo and user desktop
- Original mcstas instrument file patched and difference highlighted in docs

### CODE REVIEW

File: Instructions/ReadMe.md [hunk 1]
```diff
@@ -34,4 +51,11 @@ the beamline with respect to the sample using the mcstas `AT (x, y, z) RELATIVE
...
 The Lambda detector is set to detect wavelengths between l_min_source and l_max_source which are
-calculated based on the instrument parameters lmin=3.0AA and lmax=9.7AA.
\ No newline at end of file
+calculated based on the instrument parameters lmin=3.0AA and lmax=9.7AA.
+
+
+### MCSTAS Team
+
+For further information contact the ESS Mcstas Team:
+    - Torben Nielsen (Torben.Nielsen@esss.se)
```
Comment (OwenArnold) [inline Instructions/ReadMe.md]: As this is a public repository, maybe not having the email address would be a good idea. Avoid this being harvested. Names should be enough, or point to their website etc.

Comment (OwenArnold) [reply Instructions/ReadMe.md]: Otherwise the text changes look good. I'll see what they look like in PDF form too.

Comment (LamarMoore) [reply Instructions/ReadMe.md]: Ok I will change that

File: Vagrantfile [hunk 2]
```diff
@@ -39,6 +39,11 @@ Vagrant.configure(2) do |config|
   
   #copy Instructions for running mcstas
   config.vm.provision "file", source: "./Instructions/ReadMe.pdf", destination: "/home/vagrant/Desktop/ReadMe.pdf"
+  #copy McStas manuals
+  config.vm.provision "file", source: "./Instructions/Manuals/", destination: "/home/vagrant/Desktop/"
   #copy LOKI Files
   config.vm.provision "file", source: "./LOKI/", destination: "/home/vagrant/Desktop/"
+  
+  config.vm.provision "shell",
```
Comment (OwenArnold) [inline Vagrantfile:47]: 👍

**Comment by LamarMoore** [comment]
@OwenArnold  made that change.

**Pull request merged**